common/transport/amqp/src/amqp.ts

```diff
@@ -135,6 +88,280 @@ export class Amqp {
         forever: false
       }
     }, amqp10.Policy.EventHub));
+
+    this._fsm = new machina.Fsm({
+      namespace: sdkVersionString,
```

Comment:
>namespace: sdkVersionString, [](start = 6, length = 28)

Interesting choice.  Is this arbitrary or does it serve a purpose?

Reply:
arbitrary during a debug phase... suggestions? could be simply "amqp" since links are "senderlink" and "receiverlink"

---
In reply to: [139500399](https://github.com/Azure/azure-iot-sdk-node/pull/83#discussion_r139500399) [](ancestors = 139500399)

Reply:
good enough.  Just curious.

---
In reply to: [139537523](https://github.com/Azure/azure-iot-sdk-node/pull/83#discussion_r139537523) [](ancestors = 139537523,139500399)